inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/FactLinkingService.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.tudarmstadt.ukp.inception.ui.kb.feature;
+
+import java.util.List;
+
+import org.apache.uima.jcas.JCas;
+
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.inception.kb.graph.KBHandle;
+
+public interface FactLinkingService
+{
```

Comment:
`FactLinkingService` currently is only a class used to reduce the duplicate code in SubjectObject/Property/Modifier editor. Maybe it shouldn't be implemented as a service, but a util. WDYT @reckart @Rentier ?

Reply:
You reference another service inside the FactLinkingService. It's better to keep it is a Spring component so you don't have to pass the kbService around as a parameter.

Btw. please use `@Autowire` instead of `@Resource`- the latter may not always work.

Reply:
changed.